apps/web-giddh/src/assets/fonts/icomoon/fonts/bkcup-3-jun/icomoon.svg

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Can we not keep this backup on server?

Reply:
we can, but every new icon we need to make backup and replace whole file of icomoon

Reply:
we need to discuss about it, please ignore for this PR iif you can. will do needful for this